Makefile

```diff
@@ -64,10 +64,13 @@ cmd/support-scheduler/support-scheduler:
 cmd/security-secrets-setup/security-secrets-setup:
 	$(GO) build $(GOFLAGS) -o ./cmd/security-secrets-setup/security-secrets-setup ./cmd/security-secrets-setup
 
+cmd/security-pki-init/security-pki-init:
+	$(GO) build $(GOFLAGS) -o ./cmd/security-pki-init/security-pki-init ./cmd/security-pki-init
+
 clean:
 	rm -f $(MICROSERVICES)
 
-test:
+test: cmd/security-secrets-setup/security-secrets-setup
```

Comment:
Would it be feasible to refactor the code in order to have a mock PKI generator that does not require the unit tests to call an actual instance of this service? This would remove the need to add this to the build target.

Reply:
Yea, we can look into that.